lib/plugins/aws/provider/awsProvider.js

```diff
@@ -151,6 +151,9 @@ class AwsProvider {
               { $ref: '#/definitions/awsCfFunction' },
             ],
           },
+          awsString: {
```

Comment:
Maybe let's name it `awsCfAddress` or `awsCfInstruction` (which is better?), at least _string_ suggests it's unconditionally _string_ (e.g. we have `awsArnString` and it's unconditionally an ARN string)

Reply:
Let's go with `awsCfInstruction`